test/integration/controller/jobs/pod/suite_test.go

```diff
@@ -73,13 +74,22 @@ var _ = ginkgo.AfterSuite(func() {
 	fwk.Teardown()
 })
 
-func managerSetup(configuration *config.Configuration, opts ...jobframework.Option) framework.ManagerSetup {
+func managerSetup(
+	setupTASControllers bool,
+	enableScheduler bool,
```

Comment:
Why do we introduce this option? It does not seem needed for this PR since the new tests pass true here.
So, I would consider it as a useful extension, but for separate PR.

Reply:
Created https://github.com/kubernetes-sigs/kueue/pull/3420.